tools/fuchsia/dart.gni

```diff
@@ -1,16 +1,13 @@
-# Copyright 2013 The Flutter Authors. All rights reserved.
+# Copyright 2018 The Fuchsia Authors. All rights reserved.
```

Comment:
never increase a date on a copyright

Reply:
oops, i reverted it to an older version of the file.